src/components/screens/timebank-screen.tsx

```diff
@@ -43,7 +43,7 @@ const TimebankScreen = () => {
    * Gets person's total time data.
    */
   const getPersonTotalTime = async () => {
-    if (persons.length) {
+    if (!persons.length) {
```

Comment:
Again the ! should have only been added to the function used in the  vacations card, using it here prevents the data being fetched for the timebank screen so the ! needs to be removed here.